### PR TITLE
Included mysql in test suite

### PR DESCRIPTION
**Pull request by CGoodwin90**
Updated test suite to include Mysql 5.7 & 8.0.

### CODE REVIEW

**Comment by tobybellwood** [comment]
@CGoodwin90  Because this uses the same driver as Mariadb and we've now split the driver from the service - this can now just be an alias path to the handler, right?

**Comment by CGoodwin90** [comment]
> @CGoodwin90 Because this uses the same driver as Mariadb and we've now split the driver from the service - this can now just be an alias path to the handler, right?

Yep! I've got that in refactor branch I've been working on, I'll pull it into here.

**Comment by tobybellwood** [comment]
Nice - I used it to make the mysql tests for 8.0 and 8.4 here (no 5.7 any more) - https://github.com/uselagoon/lagoon-images/pull/308/files#diff-221bd0bdc17f3585dd7d2ba3aecbc805087e528fb3d913edd655a7d01f8494ddR252-R253

File: TESTING_dockercompose.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mariadb?servic
 docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mariadb?service=mariadb-10-11" | grep "SERVICE_HOST=10.11"
 docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mariadb?service=mariadb-10-11" | grep "LAGOON_TEST_VAR=internal-services-test"
 
+# mysql-8-0 should be able to read/write data
+docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mysql?service=mysql-8-0" | grep "SERVICE_HOST=8.0.34"
```
Comment (tobybellwood) [inline TESTING_dockercompose.md:49]:
```suggestion
docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mysql?service=mysql-8-0" | grep "SERVICE_HOST=8.0"
```

File: TESTING_dockercompose.md [hunk 2]
```diff
@@ -45,6 +45,14 @@ docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mariadb?servic
...
+docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mysql?service=mysql-8-0" | grep "LAGOON_TEST_VAR=internal-services-test"
+
+# mysql-8-4 should be able to read/write data
+docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mysql?service=mysql-8-4" | grep "SERVICE_HOST=8.4.0"
```
Comment (tobybellwood) [inline TESTING_dockercompose.md:53]:
```suggestion
docker compose exec -T commons sh -c "curl -kL http://go-web:3000/mysql?service=mysql-8-4" | grep "SERVICE_HOST=8.4"
```

File: docker-compose.yml [hunk 3]
```diff
@@ -57,6 +57,20 @@ services:
     ports:
       - '5432'
 
+  mysql-8-0:
+    image: testlagoon/mysql-8.0:experimental
```
Comment (tobybellwood) [inline docker-compose.yml:61]:
```suggestion
    image: testlagoon/mysql-8.0:pr-308
```

**Review by tobybellwood — APPROVED** [review]
nice and easy, good job

**Pull request merged**